lib/tasks/mailers.rake

```diff
@@ -56,4 +56,9 @@ namespace :mailers do
   task send_purge_warning: :environment do
     SendPurgeWarningsJob.perform_later
   end
+  
+  desc "Sends patient digest to users"
```

Comment:
Make sure to add a section to the README about this new job: https://github.com/SaraAlert/SaraAlert/blame/master/README.md#L136